test/key_value_SUITE.erl

```diff
@@ -9,6 +9,23 @@ all() ->
    key_value_test,
    coverage_test].
 
+-if(?OTP_RELEASE >= 25).
+init_per_suite(Config) ->
```

Comment:
some comment here explaining why otp 25 needs special setup would be good

Reply:
yes, this is strange. what is the reason behind this?

Reply:
The reason behind this is that the ct_slave module is deprecated in OTP 25, and will be removed in OTP 27.
In its place, I have to use the peer module to start a node and run the tests.